.travis.yml

```diff
@@ -9,16 +9,24 @@ php:
   - hhvm
   - nightly
 
+env:
+- NO_STYLE=1
```

Comment:
Appears that removing it actually sets the variable
<img width="812" alt="screen shot 2017-04-10 at 21 27 21" src="https://cloud.githubusercontent.com/assets/3288888/24881085/9510fa56-1e34-11e7-9038-f4e8cd17161d.png">
<img width="512" alt="screen shot 2017-04-10 at 21 27 45" src="https://cloud.githubusercontent.com/assets/3288888/24881084/950fc974-1e34-11e7-988e-9a5297888e59.png">